cpp/modmesh/pilot/RWorld.hpp

```diff
@@ -50,33 +50,41 @@
 namespace modmesh
 {
 
-/**
- * Make a world viewable.
- */
-class RWorld
+class RVertices
     : public Qt3DCore::QEntity
 {
+    Q_OBJECT
```

Comment:
I forgot to add `Q_OBJECT` for `RWorld`.  Now add it to `RVertices` and `RLines`.